angular08thOct/src/app/alertme/alertme.component.html

```diff
@@ -0,0 +1,9 @@
+<p>alertme works!</p>
+<html>
+<head>
```

Comment:
we dont have to give head and body, we can directly use tags

Reply:
ok